common/src/main/resources/docker/ecs.bom

```diff
@@ -0,0 +1,231 @@
+          type: boolean
+          default: true
+        - name: ecs.cluster.name
+          label: "ECS Cluster Name"
+          description: |
+            The name of the ECS cluster
+          type: string
+          default: "clocker"
+
+      brooklyn.children:
+        - type: docker-cluster
+          id: docker-cluster
+          name: "docker-cluster"
+
+  - id:  docker-cluster
+    name: "Docker Cluster"
+    description: |
+      Creates a cluster of Docker engines, of configurable initial size, and
+      configures them for use with the Amazon EC2 comtainer service.
+    itemType: entity
+    item:
+      type: cluster
+
+      brooklyn.parameters:
+        - name: docker.initial.size
+          label: "Initial Cluster Size"
+          description: |
+            Size of the Docker cluster when created initially
+          type: integer
+          default: 1
+        - name: docker.max.size
+          label: "Maximum Cluster Size"
+          description: |
+            Maximum size the Docker cluster can be scaled to
+          type: integer
+          default: 5
+        - name: docker.scaling.cpu.limit
+          label: "Docker Scaling CPU Limit"
+          description: |
+            The average CPU usage limit for the Docker cluster, before another node
+            will automatically be added. The default is 0.95 or 95%
+          type: double
+          default: 0.95
+        - name: docker.recovery.quarantineFailedEntities
+          label: "Quarantine"
+          description: |
+            Quarantine failed entities instead of destroying them
+          type: boolean
+          default: true
+        - name: docker.recovery.failOnRecurringFailuresInThisDuration
+          label: "Fail Duration"
+          description: |
+            Reports entity as failed if it fails two or more times in this time window
+          type: long
+          default: 300000
+
+      brooklyn.policies:
+        - type: org.apache.brooklyn.policy.ha.ServiceReplacer
+          brooklyn.config:
+            failureSensorToMonitor: $brooklyn:sensor("ha.entityFailed")
+            failOnRecurringFailuresInThisDuration:
+              $brooklyn:config("docker.recovery.failOnRecurringFailuresInThisDuration")
+        - type: org.apache.brooklyn.policy.autoscaling.AutoScalerPolicy
+          brooklyn.config:
+            autoscaler.metric:
+              $brooklyn:sensor("org.apache.brooklyn.entity.machine.MachineAttributes", "cpu.average")
+            autoscaler.metricLowerBound: 0.00
+            autoscaler.metricUpperBound:
+              $brooklyn:config("docker.scaling.cpu.limit")
+            autoscaler.minPoolSize:
+              $brooklyn:config("docker.initial.size")
+            autoscaler.maxPoolSize:
+              $brooklyn:config("docker.max.size")
+            autoscaler.resizeUpStabilizationDelay: 30s
+            autoscaler.resizeDownIterationMax: 0 # Disable scaling down
+            autoscaler.resizeDownStabilizationDelay: forever
```

Comment:
Will this cause us to track an infinite history of cluster load, so that we try to calculate what load has been happening "forever"? (I haven't dug into the Brooklyn `AutoScalerPolicy` to check; if you're going to rely on this, then perhaps we should have a test in Brooklyn? Or maybe we should just add an explicit `autoscaler.resizeDownEnabled: false`).